invenio_oauthclient/handlers/token.py

```diff
@@ -177,7 +210,16 @@ def token_getter(remote, token=""):
         # Store token and secret in session
         session[session_key] = remote_token.token()
 
-    return session.get(session_key, None)
+    values = session.get(session_key, None)
+    if values:
+        # Continue supporting the old tuple for backwards-compatibility with existing sessions
+        if len(values) == 2:
```

Comment:
does this mean we plan to support old style tokens indefinitely or we plan to phase them out at some point?